assets/css/style.css

```diff
@@ -194,11 +204,180 @@ body {
     border-top: solid 2px var(--dark-orange-glow);
     box-shadow: 0 0 10px 5px var(--dark-orange-glow);
     margin-top: 20px;
-    padding-bottom: 15px;
 }
 
-.github-link {
-    margin-bottom: 20px;
+.footer-text {
+    text-align: center;
+    font-family: niveau-grotesk, sans-serif;
+    font-weight: 400;
+    font-style: italic;
+    color: var(--text-color);
+}
+
+.footer-subtext {
+    text-align: center;
+    margin-top: 15px;
+}
+
+.columns {
+    margin-left: 10%;
+    margin-top: 2%;
+}
+
+/* search bar design functionality */
+
```

Comment:
You could create classes specific to the search bar rather than staying broad with .container, .box, *, etc.. This will help prevent overwriting CSS styling elsewhere on the page. Even if it looks okay with these changes currently, it is not very efficient and holds us back when adding more features and pages in the future.